scheduler/scheduler.go

```diff
@@ -589,11 +590,19 @@ func (s *scheduler) gatherMetricsAndPlugins(wf *schedulerWorkflow) ([]core.Metri
 	)
 
 	for _, m := range wf.metrics {
-		mts = append(mts, &metric{
-			namespace: m.Namespace(),
-			version:   m.Version(),
-			config:    wf.configTree.Get(m.Namespace()),
-		})
+		nss, err := s.metricManager.ExpandWildCards(m.Namespace())
+		if err != nil {
+			// use metric directly from the workflow
+			nss = [][]string{m.Namespace()}
+		}
+
+		for _, ns := range nss {
+			mts = append(mts, &metric{
+				namespace: ns,
+				version:   m.Version(),
+				config:    wf.configTree.Get(m.Namespace()),
```

Comment:
Loading mock1 and running the following task fails.
```
---
  version: 1
  schedule:
    type: "simple"
    interval: "1s"
  workflow:
    collect:
      metrics:
        /intel/*: {}
      config:
        /intel/mock:
          user: "root"
          password: "secret"
``` 

To fix this line should be `config:    wf.configTree.Get(ns),`.